app/views/stations/dashboard.erb

```diff
@@ -0,0 +1,51 @@
+<% @title = "Station Dashboard" %>
+
+<div class="jumbotron">
+<h1 class="page-header">Station Dashboard
+    <a id="edit-large" class="btn btn-warning btn-small" href="/stations" role="button">Station Index</a></h1>
+
+<ul>
+<li id="stations-t">Total count of stations: <%= Station.total_station_count %></li>
```

Comment:
we also talked about this in your evaluation. we want to create a PORO or possibly a method on `Station` that grabs all of our data in a nice way so we can simply call the variable here rather than directly accessing the model. by directly accessing the model in our view, we're overstepping our controller